pkgs/by-name/le/lenmus/package.nix

```
@@ -1,37 +1,42 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, cmake
-, pkg-config
-, makeWrapper
-, boost
-, portmidi
-, sqlite
-, freetype
-, libpng
-, pngpp
-, zlib
-, wxGTK32
-, wxsqlite3
-, fluidsynth
-, fontconfig
-, darwin
-, soundfont-fluid
-, openlilylib-fonts
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  makeWrapper,
+  boost,
+  portmidi,
+  sqlite,
+  freetype,
+  libpng,
+  pngpp,
+  zlib,
+  wxGTK32,
+  wxsqlite3,
+  fluidsynth,
+  fontconfig,
+  darwin,
+  soundfont-fluid,
+  openlilylib-fonts,
 }:
 
 let
   inherit (darwin.apple_sdk.frameworks) Cocoa;
 in
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
FYI [fixed point makes eval times slower](https://github.com/NixOS/nixpkgs/issues/320528#issuecomment-2175087036). My suggestion is to only make such change when it's a necessity.

---

Most packages use "rec". Which is cheap computation wise. The push for a fixed point replacement is a debt to be paid in eval times. If you think this is wise. Go ahead. I won't stop you.